RELEASING.md

```diff
@@ -0,0 +1,264 @@
+# Versioning and release processes
+
+## Versioning Process
+
+Ledger team has fully adopted the [PVP versioning scheme](https://pvp.haskell.org/) for
+all packages in this repo.
+
+Updating the version for every package in this repository is the responsibility of the
+developer that is introducing a change to any package that is subject to release
+process. Version must be updated in both the `CHANGELOG.md` and in the `.cabal` file for
+the package in the same PR, as well as every other package that is affected by the change
+in the repository. Every affected package that is subject to release process must receive
+a lower and, potentially, an upper bound update on the dependency that has experienced the
+change.
+
+For example if I add a function to `cardano-ledger-core` and use it in
+`cardano-ledger-conway`, then I must add a minor version bump to `cardano-ledger-core` and
+I also must add a lower version restriction in `cardano-ledger-conway` for
+`cardano-ledger-core`, as per PVP.
+
+In other words, when it comes to versioning, the fact that all of the packages in this
+repository are build together at the same time should be ignored. It should be assumed
```

Comment:
Just a little nitpick:  `are built together`

Reply:
That's not a nitpick but a legit spelling mistake. :slightly_smiling_face: Thank you.